src/interfaces/userInterface.ts

```diff
@@ -6,4 +6,5 @@ export interface User {
   salary?: number;
   projects?: string[];
   imageUrl?: string;
+  status?: string;
```

Comment:
Is this property really optional? Because I would say that the user would ALWAYS have a status, either active or inactive.